xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -73,6 +78,34 @@ final class WeightedRoundRobinLoadBalancer extends RoundRobinLoadBalancer {
   private final AtomicInteger sequence;
   private final long infTime;
   private final Ticker ticker;
+  private static final LongCounterMetricInstrument rrFallbackCounter;
```

Comment:
Let's not mix static and non-static fields. Move these up next to the logger.

These are immutable, so should use UPPER_CASE.